hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/SimpleRpcSchedulerFactory.java

```diff
@@ -37,13 +37,29 @@ public class SimpleRpcSchedulerFactory implements RpcSchedulerFactory {
   public RpcScheduler create(Configuration conf, PriorityFunction priority, Abortable server) {
     int handlerCount = conf.getInt(HConstants.REGION_SERVER_HANDLER_COUNT,
       HConstants.DEFAULT_REGION_SERVER_HANDLER_COUNT);
-    return new SimpleRpcScheduler(conf, handlerCount,
-      conf.getInt(HConstants.REGION_SERVER_HIGH_PRIORITY_HANDLER_COUNT,
-        HConstants.DEFAULT_REGION_SERVER_HIGH_PRIORITY_HANDLER_COUNT),
-      conf.getInt(HConstants.REGION_SERVER_REPLICATION_HANDLER_COUNT,
-        HConstants.DEFAULT_REGION_SERVER_REPLICATION_HANDLER_COUNT),
-      conf.getInt(HConstants.MASTER_META_TRANSITION_HANDLER_COUNT,
-        HConstants.MASTER__META_TRANSITION_HANDLER_COUNT_DEFAULT),
-      priority, server, HConstants.QOS_THRESHOLD);
+    boolean bulkLoadIndependenceQueueEnabled =
```

Comment:
Do we need a separated config to enable this feature? I suppose we just set the handler count to 0 is enough?